dockerfiles/hpcc/templates/_util.tpl

```diff
@@ -29,10 +29,10 @@ apiVersion: v1
 metadata:
   name: {{ .me.name }}-configmap 
 data:
-  global.json: |
-    {
-      "version": {{ .root.Values.global.image.version | quote }}
-    }
+  global.yaml: |
+    version: "1.0"
+    Global:
+      imageVersion: {{ .root.Values.global.image.version | quote }}
 {{ include "hpcc.utils.generateConfigMapFromFile" . | indent 2 }}
```

Comment:
indent 4?

Reply:
2 is correct I think

Reply:
yes think it's supposed to be aligned with global.yaml

Reply:
It's creating something at the same level as "global.yaml" above

Reply:
generateConfigMapFromFile is not very well named - it's generating config file from inline or file

Reply:
ok